### PR TITLE
Survey stats docs

### PR DESCRIPTION
**Pull request by bnelo12**

### CODE REVIEW

File: source/includes/demand/_statistics.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+#### Sessions Model
+
+| Property                     | Type     | Description                                                                                |
+|------------------------------|----------|--------------------------------------------------------------------------------------------|
+| fulcrumStatus                | int      | The status code returned by Fulcrum.                                                       |
+| clientStatus                 | int      | The status code returned by the client.                                                    |
+| entryDate                    | datetime | The date-time when the session started.                                                    |
+| lastDate                     | datetime | The date-time when the session ended.                                                      |
+| supplierID                   | int      | The supplierID of the supplier that provided the entrant                                   |
+| PID                          | string   | Panelist ID.                                                                               |
```
Comment (brenankeller) [inline source/includes/demand/_statistics.md]: @wbbradley @bnelo12 @nikheintzen To keep property definitions consistent across the documentation, we always use the previous definition if a term is already defined. We put together a glossary at one point to keep terms unified. You can also search the documentation. The glossary is here: https://docs.google.com/spreadsheets/d/18YdyAdwOtcSVcV9GArMEeRml8NQqrnXVZTtWdh23-ew/edit#gid=0

For PID the definition is: A supplier's unique respondent identifier.

File: source/includes/demand/_statistics.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+| RID                          | string   | Respondant ID.                                                                             |
```
Comment (brenankeller) [inline source/includes/demand/_statistics.md]: RID has not been returned anywhere in the API before so we don't have a description, but we typically try to provide context to the developers about what the property is. From the stype guide in the README: Description: 1-2 sentence description. Use terse statements (Do not start with words such as “This”, “Returns”, etc). End with a period. Avoid using the exact same words as the property.

File: source/includes/demand/_statistics.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
+##Statistics
+
+The Statistics resource allows the buyer to view valuable survey performance statistics based on the supplier and timeframe extending up to three months in the past. These survey stats provide the data necessary for a Fulcrum buyer to determine the health of their survey, including: number of entrants, aggregate data on Fulcrum and client response codes, cost of the survey, cost per an entrant, system conversion, and median length of interview.
+
+#### Sessions Model
+
+| Property                     | Type     | Description                                                                                |
+|------------------------------|----------|--------------------------------------------------------------------------------------------|
+| fulcrumStatus                | int      | The status code returned by Fulcrum.                                                       |
```
Comment (brenankeller) [inline source/includes/demand/_statistics.md]: This doesn't match the capitalization of properties across the documentation. i.e. `StudyTypeID` If we were to change this style we need to roll it out across the site.

File: source/includes/demand/_statistics.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+  ]
+}
+```
+
+#### Query Arguments
+
+| Property                     | Type     | Required  | Description                                                                    |
+|------------------------------|----------|-----------|--------------------------------------------------------------------------------|
+| Survey Number                | int      | true      | Unique ID associated with the survey.                                          |
+| Entry After                  | datetime | false     | The date and time after an entrant started a survey.                           |
```
Comment (brenankeller) [inline source/includes/demand/_statistics.md]: Spaces in the property name?

File: source/includes/demand/_statistics.md (hunk 2 above)
Comment (nikheintzen) [inline source/includes/demand/_statistics.md]: Respondant (spelling)

File: source/includes/demand/_statistics.md (hunk 4 above)
Comment (bnelo12) [inline source/includes/demand/_statistics.md]: @brenankeller How would you like query arguments implemented?

File: source/includes/demand/_statistics.md (hunk 1 above)
Comment (wbbradley) [inline source/includes/demand/_statistics.md]: Should we put that glossary into the docs?

Comment (brenankeller) [inline source/includes/demand/_statistics.md]: Probably so.

**Pull request closed without merging**